packages/tychon/data_stream/volume/elasticsearch/ingest_pipeline/rest.yml

```
@@ -0,0 +1,56 @@
+---
+description: Pipeline for TYCHON Volumes
+processors:
+
+  - set:
+      tag: set_tychon_volume_block_size
+      field: tychon.volume.block_size
+      value: 0
+      if: ctx.tychon.volume?.block_size == null
+  - set:
+      tag: set_tychon_volume_freespace
+      field: tychon.volume.freespace
+      value: 0
+      if: ctx.tychon.volume?.freespace == null
+  - set:
+      tag: set_tychon_volume_percent_full
+      field: tychon.volume.percent_full
+      value: 100
+      if: ctx.tychon.volume?.percent_full == null || ctx.tychon.volume?.percent_full == 'NaN'
```

```suggestion
      if: ctx.tychon.volume?.percent_full == null || ctx.tychon.volume.percent_full == 'NaN'
```
This should short-circuit, so if the first branch is true, the second will not be evaluated.

---

Done.